core/storage/classifier/gae_models.py

```diff
@@ -171,17 +173,17 @@ def create(
         return instance_id
 
     @classmethod
-    def query_new_and_pending_training_jobs(cls, cursor=None):
+    def query_new_and_pending_training_jobs(cls, offset=0):
```

Comment:
Is it ok to not specify a default value here? Would prefer that if possible.

Reply:
I confirmed that there are no cases where the default value is being used. Done.